x/incentive/keeper/keeper_fees_test.go

```diff
@@ -15,24 +17,80 @@ func TestCollectGasFeesToIncentiveModule(t *testing.T) {
 	app := simapp.InitElysTestApp(initChain)
 	ctx := app.BaseApp.NewContext(initChain, tmproto.Header{})
 
-	ik, bk := app.IncentiveKeeper, app.BankKeeper
+	ik, bk, amm := app.IncentiveKeeper, app.BankKeeper, app.AmmKeeper
 	// Collect gas fees
 	collectedAmt := ik.CollectGasFeesToIncentiveModule(ctx)
 
 	// rewards should be zero
 	require.True(t, collectedAmt.IsZero())
 
 	// Generate 1 random account with 1000stake balanced
-	addr := simapp.AddTestAddrs(app, ctx, 1, sdk.NewInt(1000))
+	addr := simapp.AddTestAddrs(app, ctx, 1, sdk.NewInt(1000000))
 	transferAmt := sdk.NewCoin(ptypes.Elys, sdk.NewInt(100))
 
-	// Deposit 100stake to FeeCollectorName wallet
+	// Deposit 100elys to FeeCollectorName wallet
 	err := bk.SendCoinsFromAccountToModule(ctx, addr[0], authtypes.FeeCollectorName, sdk.NewCoins(transferAmt))
 	require.NoError(t, err)
 
+	// Create a pool
+	// Mint 100000USDC
+	usdcToken := sdk.NewCoins(sdk.NewCoin(ptypes.USDC, sdk.NewInt(100000)))
+
+	err = app.BankKeeper.MintCoins(ctx, types.ModuleName, usdcToken)
+	require.NoError(t, err)
+	err = app.BankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, addr[0], usdcToken)
+	require.NoError(t, err)
+
+	var poolAssets []ammtypes.PoolAsset
+	// Elys
+	poolAssets = append(poolAssets, ammtypes.PoolAsset{
+		Weight: sdk.NewInt(50),
+		Token:  sdk.NewCoin(ptypes.Elys, sdk.NewInt(100000)),
+	})
+
+	// USDC
+	poolAssets = append(poolAssets, ammtypes.PoolAsset{
+		Weight: sdk.NewInt(50),
+		Token:  sdk.NewCoin(ptypes.USDC, sdk.NewInt(10000)),
+	})
```

Comment:
Could this pool assets initialized at the variable initialization step rather than doing append operation?

Reply:
Okay let me check.